### PR TITLE
Ewl 469 create datalayer component

### PR DESCRIPTION
**Pull request by ra-phael**

### CODE REVIEW

File: src/components/DataLayer/index.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+describe('<DataLayer />', () => {
+  it('renders with expected snapshot', () => {
+    expect(render(<DataLayer dataLayer={dataLayerMock.dataLayer} />)).toMatchSnapshot();
+  });
+
+  it('renders with expected dataLayer props', () => {
+    // eslint-disable-next-line no-unused-vars
+    const wrapper = mount(<DataLayer dataLayer={dataLayerMock.dataLayer} />);
+    expect(dataLayerMock.dataLayer).toMatchObject(window.dataLayer[0]);
```
Comment (val3ntin) [inline src/components/DataLayer/index.test.js]: I think you expect `window.dataLayer[0]` to match your `dataLayerMock.dataLayer`
please switch to:

```
expect(window.dataLayer[0]).toMatchObject(dataLayerMock.dataLayer);
```

File: src/components/DataLayer/index.stories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { object } from '@storybook/addon-knobs';
+import { DataLayer } from '../..';
+import dataLayerMock from './mocks/dataLayerMock';
+
+const datalayerStories = storiesOf('DataLayer', module).addDecorator(withInfo);
+
+datalayerStories.add('configurable', () => {
+  window.__cmp = () => {};
```
Comment (val3ntin) [inline src/components/DataLayer/index.stories.js]: I don't see a quantcast usage somewhere here. maybe remove this mock ? 
or there was already quantcast integrated ?

File: src/components/DataLayer/index.test.js (hunk 1 above)
Comment (ra-phael) [reply src/components/DataLayer/index.test.js]: Well spotted 👍

File: src/components/DataLayer/index.stories.js (hunk 2 above)
Comment (ra-phael) [reply src/components/DataLayer/index.stories.js]: Indeed, I don't think it's necessary. I'll remove line 12.

**Pull request merged**